### PR TITLE
Th/styles fixes continued

### PR DESCRIPTION
**Pull request by thudsonbu**
Simplified the styles a lot and cleaned up a bit. We definitely have some css that serves no function so we will need to remove that as well.

### CODE REVIEW

File: www/components/Hr/Hr.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import styles from './hr.module.css';
+
+const Hr = () => {
+  return <hr className={styles.hr} />;
+};
+
+export default Hr;
```
Comment (zbarnz) [inline www/components/Hr/Hr.js:7]: Can you explain this? Why not throw a border onto existing elements?

File: www/components/CardBoard/CardBoard.module.css [hunk 2]
```diff
@@ -7,4 +7,5 @@
 
 .card_container {
   margin: .5em 0 .5em 0;
+
```
Comment (zbarnz) [inline www/components/CardBoard/CardBoard.module.css:10]: Before you push your commits you should be asking yourself if changes like these are really necessary to push along with it. I am **VERY** dissapointed

**Review by zbarnz — COMMENTED** [review]
I like the logo, where did you get that from?

File: www/styles/Shared.module.css [hunk 3]
```diff
@@ -55,13 +63,13 @@
...
 .page::-webkit-scrollbar-corner {
   background: var(--background);
-}
+} */
 
 /* thumb refers to the bar on the slider */
-.page::-webkit-scrollbar-thumb {
-  background-image: linear-gradient(var(--agenda), var(--agendaThree));
+/* .page::-webkit-scrollbar-thumb {
+  background-image: linear-gradient(var(--), var(--primary));
```
Comment (zbarnz) [inline www/styles/Shared.module.css:70]: Are we not doing the scroll bar?

**Review by zbarnz — APPROVED** [review]
Nice

File: www/components/CardBoard/CardBoard.module.css (hunk 2 above)
Comment (thudsonbu) [reply www/components/CardBoard/CardBoard.module.css:10]: I'm sorry this won't happen again 🖖🏻.

File: www/components/Hr/Hr.js (hunk 1 above)
Comment (thudsonbu) [reply www/components/Hr/Hr.js:7]: It's just a dividing line. We could just put a border for sure but sometimes it is better to create a separate element to make it clearer where the border is coming from in the markup.

File: www/styles/Shared.module.css (hunk 3 above)
Comment (thudsonbu) [reply www/styles/Shared.module.css:70]: Yeah we can, we just need to re-adjust it. It was causing some issues with the styles that I didn't want to fix immediately.

**Pull request merged**